api/internal/plugins/utils/utils_test.go

```diff
@@ -89,7 +89,8 @@ func TestUpdateResourceOptions(t *testing.T) {
 		name := fmt.Sprintf("test%d", i)
 		err := in.Append(makeConfigMap(rf, name, c.behavior, c.hashValue))
 		require.NoError(t, err)
-		err = expected.Append(makeConfigMapOptions(rf, name, c.behavior, !c.needsHash))
+		config := makeConfigMap(rf, name, c.behavior, c.hashValue)
```

Comment:
it seems you have replaced the `makeConfigMapOptions` with `makeConfigMap` function which is causing a linter error.

Additionally this is eliminating the test for  `makeConfigMapOptions` function.

Could you fix this.

Reply:
@DiptoChakrabarty Thank you so much for your review.I will fix this.